.cargo/config.toml

```diff
@@ -11,7 +11,7 @@ rustflags = [
   # "-Z",
   # "trap-unreachable=no",
   "-C",
-  "llvm-args=--inline-threshold=5",
```

Comment:
Removing this leads to warnings during compilation `warning: the `-Cinline-threshold` flag is deprecated and does nothing (consider using `-Cllvm-args=--inline-threshold=...`)`, see https://github.com/probe-rs/flash-algorithm-template/issues/13.

Reply:
Inlining leads to crash. I removed it.